lib/ui.rb

```diff
@@ -49,6 +49,8 @@ def kill_pager
       end
 
       def print(output)
+        reset_line if show_progress
```

Comment:
Ok, I'll rename the method. It's not true that the line is always reset, the check whether it's necessary or not is inside of `reset_line`.